src/local/save.go

```diff
@@ -32,7 +34,8 @@ func (m *Manager) SaveModel(record manifest.Record, source model.ValueWithKey) e
 	}
 
 	// Transform
-	if err := m.transformOnSave(record, source); err != nil {
+	configContent, err := m.transformOnSave(record, source)
```

Comment:
Pri ukladani lokalneho stavu sa vola `transformOnSave`.
Vracia to modifikovany `config.json` ... v tomto pripade odstranujeme z JSON kluc `parameters.blocks` .. kedze jednotlive blocks su samostatne adresare na disku.